components/script/animations.rs

```diff
@@ -97,6 +102,7 @@ impl Animations {
         }
 
         self.unroot_unused_nodes(&sets);
+        self.update_running_animations_presence(window, None);
```

Comment:
I'm not totally sure I understand the purpose of this addition. Is it addressing a particular issue?